packages/components/nodes/chatmodels/ChatGoogleGenerativeAI/ChatGoogleGenerativeAI.ts

```diff
@@ -74,6 +76,72 @@ class GoogleGenerativeAI_ChatModels implements INode {
                 step: 0.1,
                 optional: true,
                 additionalParams: true
+            },
+            {
+                label: 'topK',
```

Comment:
Can we change the label to be more intuitive ? and add a description as well : _'Decode using top-k sampling: consider the set of top_k most probable tokens. Must be positive'_

Reply:
Name it as `Top Next Highest Probability Tokens`?